providers/azureblob/src/main/java/org/jclouds/azureblob/AzureBlobClient.java

```diff
@@ -415,7 +415,8 @@ BlobProperties getBlobProperties(
    @PUT
    @Path("{container}/{name}")
    @QueryParams(keys = { "comp" }, values = { "metadata" })
-   void setBlobMetadata(
+   @ResponseParser(ParseETagHeader.class)
+   String setBlobMetadata(
```

Comment:
Theoretically this represents an interface change although practically it does not.